hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacityScheduler.java

```diff
@@ -677,11 +677,13 @@ private void nodeUpdate(NodeManager nm) {
   public void testMaximumCapacitySetup() {
     float delta = 0.0000001f;
     CapacitySchedulerConfiguration conf = new CapacitySchedulerConfiguration();
-    assertEquals(CapacitySchedulerConfiguration.MAXIMUM_CAPACITY_VALUE,conf.getNonLabeledQueueMaximumCapacity(A),delta);
+    assertEquals(CapacitySchedulerConfiguration.MAXIMUM_CAPACITY_VALUE,
+            conf.getNonLabeledQueueMaximumCapacity(new QueuePath(A)), delta);
     conf.setMaximumCapacity(A, 50.0f);
-    assertEquals(50.0f, conf.getNonLabeledQueueMaximumCapacity(A),delta);
+    assertEquals(50.0f, conf.getNonLabeledQueueMaximumCapacity(new QueuePath(A)), delta);
```

Comment:
I think these fields should be QueuePath objects:
```
  public static final String A = CapacitySchedulerConfiguration.ROOT + ".a";
  public static final String B = CapacitySchedulerConfiguration.ROOT + ".b";
  public static final String A1 = A + ".a1";
  public static final String A2 = A + ".a2";
  public static final String B1 = B + ".b1";
  public static final String B2 = B + ".b2";
  public static final String B3 = B + ".b3";
  public static final float A_CAPACITY = 10.5f;
  public static final float B_CAPACITY = 89.5f;
  public static final String P1 = CapacitySchedulerConfiguration.ROOT + ".p1";
  public static final String P2 = CapacitySchedulerConfiguration.ROOT + ".p2";
  public static final String X1 = P1 + ".x1";
  public static final String X2 = P1 + ".x2";
  public static final String Y1 = P2 + ".y1";
  public static final String Y2 = P2 + ".y2";
```
Rather than making a QueuePath object from "A" all the time.

Reply:
It is not necessary to change the type all of it. And it isn't worth to change type A. The only occurrence of A in this class is in this method.